tests/e2e/framework/multicluster.go

```diff
@@ -46,3 +52,85 @@ func getKubeConfigFromFile(dirname string) (string, error) {
 	}
 	return remoteKube, nil
 }
+func (k *KubeInfo) createRemoteSecrets() error {
+	var err error
+	var caSecret *v1.Secret
+	// need to give time for the secret to be created on the local cluster
+	for i := 0; i <= 40; i++ {
+		caSecret, err = k.KubeClient.CoreV1().Secrets(k.Namespace).Get("istio.default", meta_v1.GetOptions{})
+		if err == nil {
+			break
+		}
+		time.Sleep(time.Second * 5)
```

Comment:
can you sleep less?  time.Second should be good enough.

Reply:
Sure - I can shorten the time to check,  but the long time is how many times through the loop.  Which based on testing I don't want to reduce much